.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+**Please check or ~strike-through~ all of the following**
+This pull-request
+- [ ] resolves #<issue-id>
+- [ ] has a meaningful title
+- [ ] complies to the [code of conduct](../CODE_OF_CONDUCT.md)
+- [ ] installs new dependencies through `npm run bootstrap`
+- [ ] contains a breaking change
+
+**Changes**
+- Your changes here...
```

Comment:
What would I write here? Summary of all the changes? Not sure if necessary, because the pull requesting person already gives a summary on github's PR page

Reply:
What files had to be changed to achieve what you've done and if you fixed something minor as well, you can add it here, too. It makes the review a bit easier